MvvmCross/Binding/iOS/Target/MvxUITextViewTextTargetBinding.cs

```diff
@@ -46,7 +46,16 @@ public override void SubscribeToEvents()
                 return;
             }
 
-            target.Changed += this.EditTextOnChanged;
+			var textStorage = target.LayoutManager?.TextStorage;
+
+			if (textStorage == null)
+			{ 
+			    MvxBindingTrace.Trace(MvxTraceLevel.Error,
+						  "Error - NSTextStorage of UITextView is null in MvxUITextViewTextTargetBinding");
+				return;
+			}
+
+            textStorage.DidProcessEditing += this.EditTextOnChanged;
```

Comment:
It is strange though, looking deeper into the documentation, it could seem like it is only deprecated for macOS: https://developer.apple.com/reference/objectivec/nsobject/1532645-textstoragedidprocessediting